### PR TITLE
Sample web page

### PR DESCRIPTION
**Pull request by calvnce**
In this project, I created; 

- HTML file with a header tag
- CSS file with a class styling
- README file
- Gitignore file
- Configured Linters for GitHub actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# hellomicroverse
+# Hello Microverse
 This is an intoductory project sponsored by Microverse for testing the students familiarirty with the basic git and github work flows and processes.
```
Comment (Temmarie) [inline README.md:2]: - [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @calvnce    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow,
- Your linter checks all pass,
- Your PR message is descriptive. 👍🏾 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️

- [ ] Kindly update your readme to be more descriptive.
- [ ] I understand your plight in fixing  the issues in your previous repo, however, for future purposes, do not open multiple Pull requests on the same project or submit an entirely new project, unless instructed to do so. If something happens and you absolutely have to, then please add the link of the previous PR in your pull request message and state why you have created a new PR/project.

_See the inline comments below for more required changes_

- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__
_Feel free to leave a comment in github if there'sanything in the review that is confusing._ :wink:

**Comment by calvnce** [comment]
This update completed the following:

-  Modified the README file to make it more descriptive

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉🎉


Congratulations! ✨🎉 @calvnce for implementing all the changes requested by the previous reviewer now am honored to approve your project its time to merge it and move on to the next project

## Optional
- Double-check your README for typo errors

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**